.github/release.yml

```
@@ -0,0 +1,5 @@
+changelog:
+  exclude:
+    authors:
+      - dependabot
+      - pre-commit-ci
```

I reckon we can also add the `github-actions` user

```suggestion
      - pre-commit-ci
      - github-actions
```